ocaml/idl/datamodel_pool.ml

```diff
@@ -1354,9 +1354,9 @@ let t =
             ~default_value:(Some (VString "")) "repository_proxy_username"
             "Username for the authentication of the proxy used in syncing with \
              the enabled repositories"
-        ; field ~in_product_since:"21.3.0" ~internal_only:true
-            ~qualifier:DynamicRO ~ty:(Ref _secret)
-            ~default_value:(Some (VRef null_ref)) "repository_proxy_password"
+        ; field ~in_product_since:"21.3.0" ~qualifier:DynamicRO
```

Comment:
I think rhe field needs to replace in_product_since with a lifecycle to mark the change

Reply:
OK. I understand it can be considered as a new field being added in this case?

Reply:
I see. Thanks. I was confused by the `in_product_since`. It seems the `lifecycle` will overwrite it, but needs a `Published` added as well.

Reply:
in_product since should be removed, the lifecycle is enough :)